net/respondd-wifi/src/respondd.c

```diff
@@ -0,0 +1,104 @@
+#include <string.h>
+#include <stdio.h>
+#include <json-c/json.h>
+#include <respondd.h>
+
+#include <linux/nl80211.h>
+
+#include "clients.h"
+#include "neighbours.h"
+#include "ifaces.h"
+
+static struct json_object *respondd_provider_statistics(void) {
+	struct json_object *result, *clients;
+	struct iface_list *ifaces;
+	int wifi24 = 0, wifi5 = 0, clients_count;
+
+	result = json_object_new_object();
+	if (!result)
+		return NULL;
+
+	clients = json_object_new_object();
+	if (!clients) {
+		json_object_put(wireless);
+		json_object_put(result);
+		return NULL;
+	}
+
+	ifaces = get_ifaces();
+	while (ifaces != NULL) {
+		if(ifaces->type != NL80211_IFTYPE_AP)
+			goto next_statistics;
```

Comment:
This would also include AP interfaces that are not used for mesh clients (for example the private-wifi interface). Another reason to only provide this as a library for the gluon-mesh-* respondd providers rather than an independent module.